explainability-core/src/test/java/org/kie/trustyai/metrics/drift/fouriermmd/FourierMMDTest.java

```diff
@@ -0,0 +1,258 @@
+
+        validDF = readCSV(validDataSetFileName);
+
+        testDF = readCSV(testDataSetFileName);
+
+    }
+
+    private Dataframe readCSV(String fileName) throws FileNotFoundException, IOException {
+
+        BufferedReader br = new BufferedReader(new FileReader(fileName));
+
+        br.readLine(); // skip header line
+
+        final List<Prediction> predictions = new ArrayList<Prediction>();
+
+        while (true) {
+
+            final String line = br.readLine();
+            if (line == null) {
+                break;
+            }
+
+            String[] values = line.split(",");
+            assert values.length == 13;
+
+            final Feature x1 = new Feature("X1", Type.NUMBER, new Value(Double.parseDouble(values[2])));
+            final Feature x2 = new Feature("X2", Type.NUMBER, new Value(Double.parseDouble(values[3])));
+            final Feature x3 = new Feature("X3", Type.NUMBER, new Value(Double.parseDouble(values[4])));
+            final Feature x4 = new Feature("X4", Type.NUMBER, new Value(Double.parseDouble(values[5])));
+            final Feature x5 = new Feature("X5", Type.NUMBER, new Value(Double.parseDouble(values[6])));
+            final Feature x6 = new Feature("X6", Type.NUMBER, new Value(Double.parseDouble(values[7])));
+            final Feature x7 = new Feature("X7", Type.NUMBER, new Value(Double.parseDouble(values[8])));
+            final Feature x8 = new Feature("X8", Type.NUMBER, new Value(Double.parseDouble(values[9])));
+            final Feature x9 = new Feature("X9", Type.NUMBER, new Value(Double.parseDouble(values[10])));
+            final Feature x10 = new Feature("X10", Type.NUMBER, new Value(Double.parseDouble(values[11])));
+
+            final List<Feature> features = new ArrayList<Feature>(
+                    Arrays.asList(x1, x2, x3, x4, x5, x6, x7, x8, x9, x10));
+
+            final PredictionInput predIn = new PredictionInput(features);
+            final PredictionOutput predOut = new PredictionOutput(new ArrayList<Output>());
+            final Prediction prediction = new SimplePrediction(predIn, predOut);
+
+            predictions.add(prediction);
+        }
+
+        br.close();
+
+        return Dataframe.createFrom(predictions);
+    }
+
+    @Test
+    void testValidData() {
+        try {
+            setup();
+
+            System.out.println(trainDF.toString());
+
+            FourierMMD fourierMMD = new FourierMMD(trainDF);
+
+            DriftResult drift = fourierMMD.calculate(validDF);
+
+            // self.assertIsNotNone(d_res)
+            // self.assertIsInstance(d_res, dict)
+            // self.assertListEqual(
+            // list(d_res.keys()), ["drift", "magnitude", "computed_values"]
+            // )
+            // self.assertFalse(d_res["drift"])
+
+            System.out.println("drift = " + drift);
+
+            assert !drift.drift;
+
+            // self.assertLess(d_res["magnitude"], 1)
+
+            assert drift.magnitude < 1.0;
```

Comment:
Please use `org.junit.jupiter.api.Assertions.*` here. See the [Meanshift Tests](https://github.com/trustyai-explainability/trustyai-explainability/blob/4dabe5409efd3ead002d83b00d154f3c634b3db9/explainability-core/src/test/java/org/kie/trustyai/metrics/drift/meanshift/MeanshiftTest.java#L566) as examples

Reply:
done